e2e/steps_file.js

```diff
@@ -502,6 +516,74 @@ module.exports = function () {
       ]);
     },
 
+    async respondToClaimSpec(responseType,defenceType,paidAmount) {
+          eventName = 'Respond to claim';
```

Comment:
As discussed, please create a follow-up ticket to address the incorrect picking of event, given we have identical names.

Reply:
created https://tools.hmcts.net/jira/browse/CIV-572 to fix this issue.